subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyGradleModuleMetadataSmokeTest.groovy

```diff
@@ -34,7 +34,7 @@ class ThirdPartyGradleModuleMetadataSmokeTest extends AbstractSmokeTest {
         useSample("gmm-example")
         // TODO Test Kotlin 1.4
         def kotlinVersion = TestedVersions.kotlin.latestStartsWith("1.3")
-        def androidPluginVersion = AGP_VERSIONS.getLatestOfMinor("4.0")
+        def androidPluginVersion = AGP_VERSIONS.getLatestOfMinor("4.1")
```

Comment:
❓ I assume the version bump is related because 4.0 would now fail?

Reply:
Correct, as wrote in the commit message